src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -185,7 +185,7 @@ export const QueryResultPane = () => {
             );
         }
         // gridCount is 1
-        return Math.max(availableHeight - TABLE_ALIGN_PX, MIN_GRID_HEIGHT);
+        return availableHeight - TABLE_ALIGN_PX;
```

Comment:
hmm, I thought I did this for the previous release (i.e. https://github.com/microsoft/vscode-mssql/pull/18292/files#diff-e6dbba4228028ff44e25b00a23fadd6fb3b01ce6004afbba3f309ff9030561deR186)?  I guess it got readded during refactoring maybe?

Reply:
Yeah, I think I accidentally re-added this at some point.